tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "moduleResolution": "node",
+    "strict": true,
+    "esModuleInterop": true
+  }
+}
```

Comment:
A `tsconfig.json` is needed for code editors to know how to configure typescript.

Reply:
Moved to #39